paystell-frontend/src/app/dashboard/layout.tsx

```
@@ -75,6 +77,14 @@ export default function DashboardLayout({ children }: { children: React.ReactNod
   }
 
   console.log('Dashboard Layout: Rendering dashboard with user');
+  
+  // End performance measurement when component is fully rendered
+  useEffect(() => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)